build/pages/about/Officers-USC.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <title>Header</title>
+
+        <!-- Custom CSS -->
+        <link rel="stylesheet" href="/build/styles/global.css" />
+        <link rel="stylesheet" href="/build/styles/announcements/announcements.css">
```

Comment:
I overlooked the pagination component. I guess we need to make it as a separate component in the future and decouple its CSS. We won't do this immediately.